src/hotspot/share/opto/reg_split.cpp

```diff
@@ -497,6 +498,7 @@ uint PhaseChaitin::Split(uint maxlrg, ResourceArea* split_arena) {
 
   uint                 bidx, pidx, slidx, insidx, inpidx, twoidx;
   uint                 non_phi = 1, spill_cnt = 0;
+  int                  delta;
```

Comment:
I think it should be local variable where it is used.

Reply:
change to local variable.